gnoblog-cli/cmd/clients.go

```diff
@@ -0,0 +1,22 @@
+package main
```

Comment:
I suggest not moving to `gnoblog-cli/cmd`. Instead, prefer either `gnoblog-cli` or `cmd/gnoblog-cli`.

The issue is that if you run `go install ./cmd`, the binary will be named `cmd`.

Reply:
Reorganized folders: [736b4b0](https://github.com/gnolang/blog/pull/63/commits/736b4b08076c898418936e0e28ec9ef583a4c8e5)